cluster/charts/crossplane/values.yaml

```
@@ -1,213 +1,218 @@
+# -- Enable `hostNetwork` for the Crossplane deployment. Caution: enabling `hostNetwork`` grants the Crossplane Pod access to the host network namespace.
 hostNetwork: false
 
-# -- Custom labels to add into metadata.
+# -- Add custom `labels` to the Crossplane pod deployment.
 customLabels: {}
 
-# -- Custom annotations to add to the Crossplane deployment and pod.
+# -- Add custom `annotations` to the Crossplane pod deployment.
 customAnnotations: {}
 
 serviceAccount:
-  # -- Custom annotations to add to the serviceaccount of Crossplane.
+  # -- Add custom `annotations` to the Crossplane ServiceAccount.
   customAnnotations: {}
 
-# -- Enable leader election for Crossplane Managers pod.
+# -- Enable leader election for the Crossplane pod.
 leaderElection: true
-# -- A list of additional args to be passed to Crossplane's container.
+# -- Add custom arguments to the Crossplane pod.
 args: []
 
 provider:
-  # -- The list of Provider packages to install together with Crossplane.
+  # -- A list of Provider packages to install.
   packages: []
 
 configuration:
-  # -- The list of Configuration packages to install together with Crossplane.
+  # -- A list of Configuration packages to install.
   packages: []
 
-# -- Names of image pull secrets to use.
+# -- The imagePullSecret names to add to the Crossplane ServiceAccount.
 imagePullSecrets: {}
 
 registryCaBundleConfig:
-  # -- Name of ConfigMap containing additional CA bundle for fetching from package registries.
-  name: {}
-  # -- Key to use from ConfigMap containing additional CA bundle for fetching from package registries.
-  key: {}
+  # -- The ConfigMap name containing a custom CA bundle to enable fetching packages from registries with unknown or untrusted certificates.
+  name: ""
+  # -- The ConfigMap key containing a custom CA bundle to enable fetching packages from registries with unknown or untrusted certificates.
+  key: ""
 
 webhooks:
-  # -- Enable webhook functionality for Crossplane as well as packages installed by Crossplane.
+  # -- Enable webhooks for Crossplane and installed Provider packages.
   enabled: true
 
 rbacManager:
-  # -- Deploy RBAC Manager and its required roles.
+  # -- Deploy the RBAC Manager pod and its required roles.
   deploy: true
-  # -- Opt out of deploying aggregated ClusterRoles.
+  # -- Don't install aggregated Crossplane ClusterRoles.
   skipAggregatedClusterRoles: false
-  # -- The number of replicas to run for the RBAC Manager pods.
+  # -- The number of RBAC Manager pod `replicas` to deploy.
   replicas: 1
-  # -- The extent to which the RBAC manager will manage permissions:.
-  # - `All` indicates to manage all Crossplane controller and user roles.
-  # - `Basic` indicates to only manage Crossplane controller roles and the `crossplane-admin`, `crossplane-edit`, and `crossplane-view` user roles.
+  # -- Defines the roles the RBAC Manager creates Roles and ClusterRoles
```

Not exactly sure what you wanted to say, maybe:
```suggestion
  # -- Defines the extent to which the RBAC Manager creates Roles and ClusterRoles
```

---

Pushed a change, do you think this is better?

---

I'm not an expert on the RBAC Manager management policies, I had a look at the code and the design doc, but I'd rather have someone with more context to judge https://github.com/crossplane/crossplane/pull/4085/files#diff-fbc48c3c51c7fab207110c647178d455b5677a3a0042592b005e849d86c46f1cR68 , maybe you can have a look at it @hasheddan?